<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelloWorldDlg</class>
 <widget class="QDialog" name="HelloWorldDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HelloWorld</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>551</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>401</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="but_runTest">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>370</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run-Test</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="but_load">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>370</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Img</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_xywh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x,y,w,h</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_color">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>550</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>color</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_xy">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>550</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x,y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_save">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>370</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_start">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>520</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_reset">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>520</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_stop">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>520</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>270</width>
       <height>200</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="ln_area">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>30</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>-1,-1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>47</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>area</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_height">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>30</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>-1,-1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_width">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>30</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>-1,-1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>47</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>width</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>47</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>height</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>47</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>kthresh</string>
      </property>
     </widget>
     <widget class="QSlider" name="sl_kthresh">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lb_kthresh">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_rect">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0,0,0,0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>rect</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>47</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>padding</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_padding">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>30</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_ratio">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>30</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0.5,1.5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>47</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>threshold</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QListWidget" name="list_funcs">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="but_live">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Live</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_log">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>520</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
